po/README.md

```diff
@@ -35,8 +34,8 @@ and commit your changes.
 ## Notes
 
 - Only commit changes for the translation file you actually edited.
-- Ignore or delete the first four lines (copyright notice) in the `$LANG.po` files.
```

Comment:
Honestly I think that line was quite nice, like the rest of the readme changes. Just the line below should be changed.

Reply:
> Honestly I think that line was quite nice, like the rest of the readme changes. Just the line below should be changed.

added it back.